src/pudl/workspace/datastore.py

```diff
@@ -440,9 +440,15 @@ def parse_command_line():
         default=False,
     )
     parser.add_argument(
-        "--populate-gcs-cache",
-        default=None,
-        help="If specified, upload data resources to this GCS bucket",
+        "--gcs-cache-path",
+        type=str,
+        help="Load datastore resources from Google Cloud Storage. Should be gs://bucket[/path_prefix]",
+    )
+    parser.add_argument(
+        "--bypass-local-cache",
+        action="store_true",
+        default=False,
+        help="If enabled, the local file cache for datastore will not be used.",
```

Comment:
Is this just a switch in how the arguments are specified? If we want to populate the GCS Cache using the script now do we give it a GCS cache path and tell it to bypass the local cache?

Reply:
Yes, If we want to populate the GCS Cache using the script now we give it a GCS cache path and also tell it to bypass the local cache.